frontend/src/app/features/in-app-notifications/entry/actors-line/in-app-notification-actors-line.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="op-ian-actors">
```

Comment:
You can ditch this div and set `@HostBinding('class.op-ian-actors')` instead. You'llprobably also need `ViewEncapsulation.None` then, but that's fine imo